ci/travis/before_install-linux.sh

```diff
@@ -47,7 +47,7 @@ python -c 'import wheel; print "wheel version=", wheel.__version__'
 while read line; do
     if [[ $line == nupic.bindings* ]];
     then
-        IFS='=' read -ra ADDR <<< "$line"
+        IFS='>=' read -ra ADDR <<< "$line"
         NUPIC_BINDINGS_VERSION="${ADDR[2]}"
```

Comment:
because IFS only works for one character, not a string, so when it was `=` it was splitting `foo==bar` into 3 strings.